modules/parquet/test/parquet-loader.spec.js

```diff
@@ -18,13 +34,71 @@ test('ParquetLoader#loader objects', (t) => {
   t.end();
 });
 
-// eslint-disable-next-line max-statements
+
 test('ParquetLoader#load', async (t) => {
   t.comment(`SUPPORTED FILES`);
+
   for (const {title, path} of SUPPORTED_FILES) {
     const url = `${PARQUET_DIR}/${path}`;
     const data = await load(url, ParquetLoader, {parquet: {url}, worker: false});
-    t.ok(data, `GOOD(${title})`);
+
+    switch (title) {
+      case 'alltypes_dictionary':
```

Comment:
This switch statement is starting to make this code rather complicated, getting hard to see what is going on.

I wonder if we should drop the `files.js` table and just load filenames directly here?